src/test/java/seedu/staffsnap/logic/commands/AddCommandTest.java

```diff
@@ -161,7 +161,7 @@ public ObservableList<Applicant> getFilteredApplicantList() {
         }
 
         @Override
-        public void updateFilteredApplicantList(Predicate<Applicant> predicate) {
+        public void updateFilteredApplicantList(Predicate<? super Applicant> predicate) {
             throw new AssertionError("This method should not be called.");
         }
 
```

Comment:
Thanks for making the generic type more flexible and fixing our unchecked warning!